Runtimes/Core/core/CMakeLists.txt

```diff
@@ -274,14 +274,13 @@ if(APPLE AND BUILD_SHARED_LIBS)
 endif()
 
 target_compile_options(swiftCore PRIVATE
+  # STAGING: Temporarily avoids having to write #fileID in Swift.swiftinterface.
+  # see also 327ea8bce2d1107a847d444651b19ca6a2901c9e
+  "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-Xfrontend -enable-experimental-concise-pound-file>"
   $<$<COMPILE_LANGUAGE:Swift>:-parse-stdlib>
   $<$<COMPILE_LANGUAGE:Swift>:-nostdimport>
   $<$<COMPILE_LANGUAGE:Swift>:-explicit-module-build>
-  $<$<COMPILE_LANGUAGE:Swift>:-disable-autolinking-runtime-compatibility-dynamic-replacements>
-  $<$<COMPILE_LANGUAGE:Swift>:-no-link-objc-runtime>
-  $<$<COMPILE_LANGUAGE:Swift>:-warn-implicit-overrides>
```

Comment:
Was this flag intentionally dropped?

Reply:
No, that has been moved to [line 177 of `Runtimes/Core/CMakeLists.txt`](https://github.com/swiftlang/swift/pull/79266/files#diff-58cf1b348980556a26ac2e00976d5dcd6d69264a438140dee54244b8f3d201f7L175) -- I double checked at desk that the resulting `build.ninja` did not drop any flags for `swiftCore`

Reply:
Bah, I failed to match up the new locations.